app/containers/SaladBowl/NounForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+
+import { appendNouns } from 'actions/root';
+
+import { css } from 'emotion';
+import { Container, Col, Row, Input, Button, Form } from 'reactstrap';
+import { useDispatch, useSelector } from 'react-redux';
+
+const NounForm = () => {
+  const dispatch = useDispatch();
+  const rootState = useSelector((state) => state);
+
+  // Local state for input
+  // TODO: the initial states on these nouns seems wrong
+  const [noun1, onNoun1Update] = useState(rootState.root.nouns[0]);
+  const [noun2, onNoun2Update] = useState(rootState.root.nouns[1]);
+  const [noun3, onNoun3Update] = useState(rootState.root.nouns[2]);
+  const [noun4, onNoun4Update] = useState(rootState.root.nouns[3]);
+  const [noun5, onNoun5Update] = useState(rootState.root.nouns[4]);
+
+  const onClick = () => {
+    const nouns = [noun1, noun2, noun3, noun4, noun5];
+    console.log('nouns submitted: ' + nouns);
+    // TODO: I would like to link the nouns submitted by a player to either their userId or the roomId
+    dispatch(appendNouns(nouns));
```

Comment:
I think likely we'll hook this action into something that triggers a call to firebase submitting the nouns, but its fine for now